templates/annuaire/index.html

```diff
@@ -28,7 +28,7 @@ <h2 class="mb-3"><a href="{% url "annuaire_home" %}">Annuaire</a></h2>
         Sélectionnez un département dans la liste pour en afficher les communes disposant d'établissements renseignés.
       </p>
       {% else %}
-      <h3 class="mt-2 mb-3">{{ current_departement.code }} - {{ current_departement.nom }}</h2>
+      <h3 class="mt-2 mb-3">{{ current_departement.code }} - {{ current_departement.nom }} ({{ communes.count }})</h3>
```

Comment:
Nope, you don't want the number of communes in the department, you want the number of erps in the departement. You'll need to create a dedicated annotation for that specific count.

Reply:
Also you'll want to format a human readable string, eg. `148 établissements` with plural handling.

Reply:
Added in https://github.com/MTES-MCT/acceslibre/pull/224/commits/9c903e4eeecf65c9a932c9541052fa54044956cc